web_digital_sign/__manifest__.py

```diff
@@ -9,11 +9,6 @@
     'depends': ['web'],
     "license": "AGPL-3",
     'category': 'Tools',
-    'description': '''
-     This module provides the functionality to store digital signature
-     Example can be seen into the User's form view where we have
-        added a test field under signature.
-    ''',
```

Comment:
We need it for SEO. Restore it back plz

Reply:
@JayVora-SerpentCS 
ok sir.i will restore it.

Reply:
@JayVora-SerpentCS 
I had updated the manifest file.